partcad/src/partcad/utils_ocp.py

```
@@ -0,0 +1,106 @@
+#   Copyright 2023 Bernhard Walter
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+
+from typing import Tuple, List
+import sys
+
+
+def extent_or_size(obj):
```

Ruff (0.8.2)

Missing return type annotation for public function `extent_or_size`

(ANN201)